### PR TITLE
Sort gems

### PR DESCRIPTION
**Pull request by DirtyF**
À la Rubocop.

/cc @jekyll/core

### CODE REVIEW

File: jekyll.gemspec [hunk 1]
```diff
@@ -29,14 +29,15 @@ Gem::Specification.new do |s|
...
-  s.add_runtime_dependency("liquid",    "~> 3.0")
-  s.add_runtime_dependency("kramdown",  "~> 1.3")
-  s.add_runtime_dependency("mercenary", "~> 0.3.3")
-  s.add_runtime_dependency("safe_yaml", "~> 1.0")
+  s.add_runtime_dependency("addressable", "~> 2.4")
   s.add_runtime_dependency("colorator", "~> 1.0")
-  s.add_runtime_dependency("rouge", "~> 1.7")
   s.add_runtime_dependency("jekyll-sass-converter", "~> 1.0")
   s.add_runtime_dependency("jekyll-watch", "~> 1.1")
+  s.add_runtime_dependency("kramdown",  "~> 1.3")
```
Comment (ashmaroli) [inline jekyll.gemspec]: `before:` the whitespace after gem-name improved readability

gemsp

`after:` it just looks weird..

gemspe

`suggestion:` **normalize whitespace and improve readability.** :smiley:

Comment (DirtyF) [inline jekyll.gemspec]: done

**Comment by quiltbag** [comment]
@jekyllbot: merge +minor

**Comment by parkr** [comment]
@quiltbag For future reference, this probably should have been `+dev` or `+site`, as it doesn't affect any of Jekyll's runtime code (`lib/`).

**Comment by quiltbag** [comment]
@parkr okay! do we have a reference for those flags somewhere?

**Comment by parkr** [comment]
@quiltbag 😄 General docs are here: http://jekyllrb.com/docs/maintaining/ and for this specifically, http://jekyllrb.com/docs/maintaining/merging-a-pull-request/#merging – no big deal, I just updated it in 22368896fb6776766e82b746fb96bb55fcbbdd0d.

**Comment by quiltbag** [comment]
@parkr thanks :o i didn't see that doc

**Comment by parkr** [comment]
@quiltbag No problem at all! Feel free to reach out with any questions.

**Pull request merged**